calc_photopionproduction.py

```diff
@@ -91,3 +84,23 @@
         header = ("Photo-pion interaction rate for the %s\n (redshift dependent)"
                   "z\tlog10(gamma)\t1/lambda_proton [1/Mpc]\t1/lambda_neutron [1/Mpc]" % field.info)
     np.savetxt(fname, data, fmt=fmt, header=header)
+
+if __name__ == "__main__":
+    fields = [
+        photonField.CMB(),
+        photonField.EBL_Kneiske04(),
+        photonField.EBL_Stecker05(),
+        photonField.EBL_Franceschini08(),
+        photonField.EBL_Finke10(),
+        photonField.EBL_Dominguez11(),
```

Comment:
I suggest we bring back the tables for the Domínguez upper/lower limit EBL. They were present a few years ago and then got lost for some reason.

Reply:
I added the upper and lower bounds for the default CRPropa data tar ball. However, I did not add them to the all the calc_* files.

Reply:
Photon field models are now imported in calc_* files. So there is only one location that needs to be updated in the future.